src/components/Input/__snapshots__/Input.spec.tsx.snap

```diff
@@ -1,150 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Input should set the htmlFor attribute for the label 1`] = `
-.c3 {
```

Comment:
It is super great that we have removed this noise, nonetheless, what do you think about having these styles for at least one test so we can audit if any detail on CSS has been updated unconsciously?

Reply:
there are a lot of snapshots still